mne/io/bti/tests/test_bti.py

```diff
@@ -195,6 +195,10 @@ def test_info_no_rename_no_reorder_no_pdf():
             np.array([ch['loc'] for ch in info['chs']]),
             np.array([ch['loc'] for ch in info2['chs']]))
 
+        # just check reading data | corner case
+        read_raw_bti(pdf_fname=pdf, config_fname=config, head_shape_fname=None,
+                     sort_by_ch_name=False, preload=True)
```

Comment:
It's a non-crash test. The bug was about preloading the data. Maybe we should test the data ordering too.

> On 19 Feb 2016, at 11:30, Alexandre Gramfort <notifications@github.com> wrote:
> 
> In mne/io/bti/tests/test_bti.py:
> 
> > @@ -195,6 +195,10 @@ def test_info_no_rename_no_reorder_no_pdf():
> >              np.array([ch['loc'] for ch in info['chs']]),
> >              np.array([ch['loc'] for ch in info2['chs']]))
> >  
> > +        # just check reading data | corner case
> > +        read_raw_bti(pdf_fname=pdf, config_fname=config, head_shape_fname=None,
> > +                     sort_by_ch_name=False, preload=True)
> you're not really testing anything right?
> 
> —
> Reply to this email directly or view it on GitHub.
>